docs/products/kafka/concepts/topic-catalog-overview.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Aiven for Apache Kafka® topic catalog
+sidebar_label: Topic catalog
+limited: true
+---
+
+The Aiven for Apache Kafka® topic catalog provides a centralized interface within the Aiven console to view and manage Apache Kafka topics across different projects and services.
+This unified view consolidates all topics into one accessible location, streamlining
+the management of your Aiven for Apache Kafka infrastructure.
+
+## Key features and benefits
+
+- **Centralized management**: Access all Apache Kafka topics across your organization
+  and projects from a single interface.
+- **Search topics**: Find specific Apache Kafka topics using the search bar.
+- **Topic information at a glance**: View topic names, associated services, projects,
+  and owners in either table view or tile view.
+
+## Governance on topic catalog
+
+With governance enabled for your organization, you can manage topic ownership and
+creation requests efficiently.
+
+- **Claim topic ownership**: You can request to claim ownership of individual topics not
+  currently owned by your team. Additionally, you can select multiple topics and submit
+  a bulk claim request for ownership transfer.
+- **Topic creation requests and approvals**: You can submit requests for creating new
+  topics, which go through an approval process to ensure proper governance.
+
```

Comment:
Optionally discover & govern topics across your organization even if you aren't a member of all projects.

?

Reply:
Added a line for this.